Source/Scene/Cesium3DTileFeature.js

```diff
@@ -92,17 +116,301 @@ define([
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    label.style = value;
+                }
+            }
+        },
+
+        /**
+         * Gets and sets the text for this feature.
+         * <p>
+         * Only applied when the feature is a point feature.
+         * </p>
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {String}
+         */
+        text : {
+            get : function() {
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    return label.text;
+                }
+                return undefined;
+            },
+            set : function(value) {
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    label.text = value;
+
+                    if (defined(value) && value !== '') {
+                        var billboard = this._billboardCollection.get(this._batchId);
+                        billboard.horizontalOrigin = HorizontalOrigin.RIGHT;
+                        label.horizontalOrigin = HorizontalOrigin.LEFT;
+                    }
+                }
             }
         }
     });
 
+    function setBillboardImage(feature) {
+        var b = feature._billboardCollection.get(feature._batchId);
+
+        if (defined(feature._billboardImage) && feature._billboardImage !== b.image) {
+            b.image = feature._billboardImage;
+            return;
+        }
+
+        if (defined(feature._billboardImage)) {
+            return;
+        }
+
+        var label = feature._labelCollection.get(feature._batchId);
+        var newColor = label.fillColor;
+        var newOutlineColor = label.outlineColor;
+        var newOutlineWidth = label.outlineWidth;
+        var newPointSize = feature._pointSize;
+
+        var currentColor = feature._billboardColor;
+        var currentOutlineColor = feature._billboardOutlineColor;
+        var currentOutlineWidth = feature._billboardOutlineWidth;
+        var currentPointSize = feature._billboardSize;
+
+        if (Color.equals(newColor, currentColor) && Color.equals(newOutlineColor, currentOutlineColor) &&
+            newOutlineWidth === currentOutlineWidth && newPointSize === currentPointSize) {
+            return;
+        }
+
+        var centerAlpha = newColor.alpha;
+        var cssColor = newColor.toCssColorString();
+        var cssOutlineColor = newOutlineColor.toCssColorString();
+        var textureId = JSON.stringify([cssColor, newPointSize, cssOutlineColor, newOutlineWidth]);
+
+        b.setImage(textureId, createCallback(centerAlpha, cssColor, cssOutlineColor, newOutlineWidth, newPointSize));
+    }
+
+    function createCallback(centerAlpha, cssColor, cssOutlineColor, cssOutlineWidth, newPixelSize) {
```

Comment:
This is from https://github.com/AnalyticalGraphicsInc/cesium/blob/4c6b691b4327a16a6065f5432d33471cddb7978a/Source/DataSources/PointVisualizer.js#L306, right?  Can we share it as a `@private` function?